app/setup.py

```diff
@@ -279,7 +279,7 @@ def setup_dynamodb(application):
 
 
 def setup_datastore(application):
-    client = datastore.Client(_use_grpc=False)
+    client = datastore.Client(_use_grpc=application.config["DATASTORE_USE_GRPC"])
```

Comment:
There is an environment variable (GOOGLE_CLOUD_DISABLE_GRPC) used by the library that we could consider using - https://github.com/googleapis/python-datastore/blob/master/google/cloud/datastore/client.py#L60

Although, that is misleading if the PubSub library ignores it 🤔

Reply:
Myself and James did discuss this env var previous;y but chose to use our own one to have finer control on which lib has grpc enabled etc.